coordinator/persistence-api/src/main/java/org/apache/cassandra/stargate/exceptions/RequestFailureReason.java

```diff
@@ -46,7 +51,7 @@ public enum RequestFailureReason {
    * between the operation and either creation or drop of the table (or, possibly, of the keyspace
    * containing that table, see comment on {@link #UNKNOWN_KEYSPACE}).
    */
-  UNKNOWN_TABLE,
+  UNKNOWN_TABLE(501),
```

Comment:
I'm not sure those numbers should become a part of the multi-version API

Reply:
Can you elaborate a bit, please? You may be right, but I want to understand why. Stargate newbie here so I will rely on the experts and I am trying not to make a mess :-)